### PR TITLE
Export capa

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: pulpcore/cli/ansible/context.py [hunk 1]
```diff
@@ -78,6 +79,8 @@ class PulpAnsibleRepositoryContext(PulpRepositoryContext):
     SYNC_ID = "repositories_ansible_ansible_sync"
     MODIFY_ID = "repositories_ansible_ansible_modify"
     VERSION_CONTEXT = PulpAnsibleRepositoryVersionContext
+    # TODO specify correct version of ansible plugin
+    CAPABILITIES = {"pulpexport": [PluginRequirement("ansible", "999.0.0")]}
```
Comment (mdellweg) [inline pulpcore/cli/ansible/context.py:83]: @daviddavis @gerrod3 do you have a proper version statement for this?

Comment (daviddavis) [reply pulpcore/cli/ansible/context.py:83]: Looks like it got added in 0.7.0:

https://github.com/pulp/pulp_ansible/blob/main/CHANGES.rst#features

**Comment by mdellweg** [comment]
The only thing this is waiting on now is the enablement of pie for pulp_container.
I can remove it and mark this as ready, WDYT?

**Review by daviddavis — APPROVED** [review]
LGTM

**Pull request merged**